test/parallel/test-whatwg-url-constructor.js

```diff
@@ -15,7 +15,8 @@ const request = {
 };
 
 /* eslint-disable */
-/* WPT Refs:
+/* The following tests are copied from WPT, modifications to them should be upstreamed first.
```

Comment:
thanks for reviewing this @gibfahn. @refack is right, the other files for test-whatwg do not contain imported code, and infact have the comment `Tests below are not from WPT.`